src/common/translations/en.json

```diff
@@ -387,7 +432,8 @@
     "done": "Done",
     "share": "SHARE",
     "check": "CHECK",
-    "update": "Update"
+    "update": "Update",
+    "accept": "ACCEPT"
```

Comment:
Needs to add this string to the other translations.

Reply:
Added